Formula/n/nvtop.rb

```diff
@@ -0,0 +1,21 @@
+class Nvtop < Formula
+  desc "Interactive GPU process monitor"
+  homepage "https://github.com/Syllo/nvtop"
+  url "https://sid.ethz.ch/debian/nvtop/macos/nvtop_3.0.2%2Bgit20240213%2Bds.orig.tar.xz"
```

Comment:
Why not this?
```suggestion
  url "https://github.com/Syllo/nvtop/archive/refs/tags/3.0.2.tar.gz"
```

Reply:
beacuse that release has no nvtop support. and it does also not include the open pr about m2